modules/ocf_admin/manifests/create/redis.pp

```diff
@@ -40,14 +38,21 @@
     notify  => Service['redis-server'];
   }
 
-  ocf::repackage { 'stunnel4': backport_on => jessie } ->
-  augeas { '/etc/default/stunnel4':
-    lens    => 'Shellvars.lns',
-    incl    => '/etc/default/stunnel4',
-    changes =>  ['set ENABLED 1'],
-  } ~>
-  file { '/etc/stunnel/redis.conf':
-    content => template('ocf_admin/stunnel/redis.conf.erb'),
-  } ~>
-  service { 'stunnel4': }
+  package { 'hitch': }
+
+  # Yes, the underscore is intentional. We already have a user named "hitch".
+  user { '_hitch':
+    groups  => 'ssl-cert',
+    notify  => Service['hitch'],
+    require => Package['hitch'],
+  }
+
+  $pem_file = "/etc/ssl/private/${::fqdn}.pem"
```

Comment:
Why not just put this in the template itself? Seems like that would be pretty reasonable and then you wouldn't need this line:

    pem-file = "/etc/ssl/private/<%= @fqdn -%>.pem"

Reply:
That's a reasonable question. At first it was a parameter to the class, but I changed my mind about generalizing the class if we're not actually planning to use redis anywhere else at the moment. I'll take your advice since it seems silly as-is.